operator/builtin/parser/syslog/syslog.go

```diff
@@ -205,6 +208,14 @@ func toBytes(value interface{}) ([]byte, error) {
 	}
 }
 
+// handleSymbols escapes characters that appear to be symbol characters, but are not.
+// If not escaped properly, go-syslog will fail to parse the entry. Quotes that are
+// escaped will not be affected.
+func handleSymbols(b []byte) []byte {
+	b = bytes.Replace(b, []byte(`\`), []byte(`\\`), -1)
+	return bytes.Replace(b, []byte(`\\"`), []byte(`\"`), -1)
```

Comment:
Am I correct in understanding that the net effect here is that all backslashes are escaped, except those that are being used to escape `"`?

Reply:
Correct. We want to leave stuff like this alone:
```
kay="some \"escaped\" value"
```